ext/mysql2/mysql2_ext.h

```diff
@@ -11,10 +11,8 @@ void Init_mysql2(void);
 
 #ifdef HAVE_MYSQL_H
 #include <mysql.h>
-#include <mysql_com.h>
 #include <errmsg.h>
 #include <mysqld_error.h>
-#include <mysql_version.h>
```

Comment:
Was added in #857 to get MariaDB 10.2 working :)

Also remove the second path below.